.github/actions/opensearch/action.yml

```diff
@@ -14,6 +14,10 @@ inputs:
     description: 'Port where you want to run the cluster'
     required: false
     default: 9200
+  disable-security:
+    description: 'Disable security plugin for the OpenSearch engine'
+    required: false
+    default: true
   opensearch_password:
```

Comment:
Where are we using this option in script? I guess we also need user name

Reply:
It isn't required. Removed.